### ISSUE TITLE
Uncaught ReferenceError: global is not defined

### ISSUE THREAD

**Issue by twobitfool** [facebook/fbjs#227]
After 396004e, started seeing the following error (in the browser): 

`Uncaught ReferenceError: global is not defined`

Perhaps we could do a quick check for `global`...
```js
doc = doc || (typeof global !== 'undefined' ? global.document : document)
```

**Comment by spicyj** [comment]
Ugh. How are you pulling in fbjs? Are you using browserify/webpack?

**Comment by twobitfool** [comment]
Browserify. 

It's not a direct dependency of the app code, but it's getting pulled in, probably as a dependency of React.

**Comment by damncabbage** [comment]
Same here; also Browserify, also using React, with fbjs not being a direct dependency.

**Comment by spicyj** [comment]
Maybe our best bet is:

```
doc = doc || (typeof document !== 'undefined' ? document : undefined);
```

@zpao What do you think?

**Comment by steadicat** [comment]
This breaks React. Any ETA on a fix?

**Comment by zpao** [comment]
Ugh, yea that seems fine. It's no worse than what we were doing before.

Interestingly, browserify (only tested latest version) wraps your module in a quick `global` shim if you're using it, so it's never `undefined`. So I'm curious if people have setups preventing that or if there's something else we should look out for… (one day `global` is going to be a real JS global and we can stop this nonsense)

```js
function(require,module,exports){
(function (global){
'use strict';
function getActiveElement(doc) /*?DOMElement*/{
  doc = doc || global.document;
  if (typeof doc === 'undefined') {
    return null;
  }
  try {
    return doc.activeElement || doc.body;
  } catch (e) {
    return doc.body;
  }
}

module.exports = getActiveElement;
}).call(this,typeof global !== "undefined" ? global : typeof self !== "undefined" ? self : typeof window !== "undefined" ? window : {})
}
```

**Comment by mjomble** [comment]
> one day global is going to be a real JS global

Interesting, could you point me towards more info on that?

**Comment by zpao** [comment]
Shipped v0.8.12.

More info on the proposal for `global`: https://github.com/tc39/proposal-global. It looks like there are some problems though… 😞
